prometheus_flask_exporter/__init__.py

```diff
@@ -413,24 +421,32 @@ def teardown_request(exception=None):
 
             response = make_response('Exception: %s' % exception, 500)
 
+            if callable(duration_group):
```

Comment:
Curious why this moved here?

Reply:
Ah, right - I forgot to update the labels (see 'path' in your above comment) - instead, the exceptions metrics should also use `group`. Since we'll always be using it, no need to have it in the conditional below. I'll update the labels